app/libs/full_contact_company_processor.rb

```diff
@@ -0,0 +1,68 @@
+class FullContactCompanyProcessor
+  def initialize(company:)
+    @company = company
+  end
+
+  def process
+    return if response.blank? || response['status'] != 200
+    fill_data
+    @company.save
+    @company
+  end
+
+  private
+
+  def links
+    @links = response['social_profiles']
+  end
+
+  def response
+    @response = call_fullcontact_api
+  end
+
+  def fill_data
+    process_links
+    process_organization
+    set_industries
```

Comment:
naming sounds weird.. you have `process_link`, `process_organisation` but `set_industries`